checks/system/unix.py

```diff
@@ -587,8 +587,8 @@ def check(self, agentConfig):
         """Return an aggregate of CPU stats across all CPUs
         When figures are not available, False is sent back.
         """
-        def format_results(us, sy, wa, idle, st):
-            data = {'cpuUser': us, 'cpuSystem': sy, 'cpuWait': wa, 'cpuIdle': idle, 'cpuStolen': st}
+        def format_results(us, sy, wa, idle, st, guest=None):
+            data = {'cpuUser': us, 'cpuSystem': sy, 'cpuWait': wa, 'cpuIdle': idle, 'cpuStolen': st, 'cpuGuest': guest}
             for key in data.keys():
                 if data[key] is None:
                     del data[key]
```

Comment:
We do want to keep `0` values, so I guess it'd rather be:
```python
data = dict((k,v) for k,v in data.iteritems() if v is not None)
```

right ?

Also I personally prefer keeping `del` as it doesn't recreate a new dict but just deletes the entry in the existing dict. But the dict is very small here so I guess it makes no difference.